src/app/shared/enums/storage.ts

```diff
@@ -16,7 +16,7 @@ export class StorageKeys {
   static SCHEDULE_TASKS = new StorageKeys('SCHEDULE_TASKS')
   static SCHEDULE_TASKS_CLINICAL = new StorageKeys('SCHEDULE_TASKS_CLINICAL')
   static SCHEDULE_REPORT = new StorageKeys('SCHEDULE_REPORT')
-  static CACHE_ANSWERS = new StorageKeys('CHACHE_ANSWERS')
+  static CACHE_ANSWERS = new StorageKeys('CACHE_ANSWERS')
```

Comment:
ok good spot. 
I don't think it was used anywhere

Reply:
I think we should keep this same as all the cached answers currently will be lost with the new storage key (as it also retrieves the cached ones using the key)